packages/payload/src/database/types.ts

```diff
@@ -77,6 +76,21 @@ export interface BaseDatabaseAdapter {
    */
   init?: Init
 
+  /**
+   * @todo make required in 4.0
+   */
+  meta?: {
```

Comment:
I don't think I would have called this `meta` should be `compatibility` or something else.